adapters/stroeerCore/stroeercore.go

```diff
@@ -60,16 +60,33 @@ func (a *adapter) MakeBids(bidRequest *openrtb2.BidRequest, requestData *adapter
 			AdM:   bid.Ad,
 			CrID:  bid.CrID,
 		}
+		bidType, _ := getMediaTypeForImp(bid.BidID, bidRequest.Imp)
 
 		bidderResponse.Bids = append(bidderResponse.Bids, &adapters.TypedBid{
 			Bid:     &openRtbBid,
-			BidType: openrtb_ext.BidTypeBanner,
+			BidType: bidType,
 		})
 	}
 
 	return bidderResponse, errors
 }
 
+func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
+	for _, imp := range imps {
+		if imp.ID == impID {
+			if imp.Video != nil {
```

Comment:
@juanmartinengo, requesting to address above comment

Reply:
@onkarvhanumante Thanks for the comment, we changed the default for banner and included MType property in Bid. We are aware that this is not ideal but we also currently do not support multiformat impression and we would defaulted to banner. When we support multiformat we will address this issue.

Reply:
How about returning an error instead of defaulting to banner.  These if checks `if imp.Banner != nil and if imp.Video != nil ` will return media type. If these check fails then bidder server has returned incorrect response





```
func getBidMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, openrtb2.MarkupType) {
	for _, imp := range imps {
		if imp.ID == impID {
			if imp.Banner != nil {
				return openrtb_ext.BidTypeBanner, openrtb2.MarkupBanner, nil
			}
			if imp.Video != nil {
				return openrtb_ext.BidTypeVideo, openrtb2.MarkupVideo, nil
			}
		}
	}
	return new BidType(""), new MarkupBanner(0), &errortypes.BadServerResponse{
		Message: fmt.Sprintf("Failed to parse impression \"%s\" mediatype", impID),
	}
}
```

Reply:
Yeah you are right, we made the change to return error and skip the bid.